src/plugins/rust/rust-parser-generator-trait.js

```diff
@@ -82,7 +82,7 @@ const RustParserGeneratorTrait = {
   buildSemanticAction(production) {
     let originalAction = this.getSemanticActionCode(production);
 
-    let {action, types} = this._extractDataTypes(originalAction);
+    let { action, types } = this._extractDataTypes(originalAction);
```

Comment:
Where this formatting is coming from? Let's use the original formatting. I'm wondering why the  https://github.com/DmitrySoshnikov/syntax/blob/master/.prettierrc with the `bracketSpacing: false` which enforces it didn't took the precedence? Perhaps you have a custom setting in your editor. Anyways, it should be `let {action, types}` per the code style.